### PR TITLE
Survey email reminder phantom filter

### LINKED ISSUE #4444: Survey email reminder
image

If I don't include Phantom students, then reminder should not be sent to them.
Maybe we can also move the reminder email button down below "Include Phantom Students"?

### PR DESCRIPTION
**Pull request by ekowidianto**
Closes #4444 

image

### CODE REVIEW

**Comment by ekowidianto** [comment]
Can be reviewed after MUI update is merged.

File: client/app/bundles/course/survey/pages/ResponseIndex/RemindButton.jsx [hunk 1]
```diff
@@ -20,7 +20,12 @@ const translations = defineMessages({
   confirmation: {
     id: 'course.surveys.ResponseIndex.RemindButton.confirmation',
     defaultMessage:
-      'Send emails to all students who have not completed the survey?',
+      'Send emails to all students (excl. phantom) who have not completed the survey?',
```
Comment (zhuhanming) [inline client/app/bundles/course/survey/pages/ResponseIndex/RemindButton.jsx:23]:
```suggestion
      'Send emails to all students (excluding phantoms) who have not completed the survey?',
```

File: client/app/bundles/course/survey/pages/ResponseIndex/RemindButton.jsx [hunk 2]
```diff
@@ -20,7 +20,12 @@ const translations = defineMessages({
...
+  },
+  confirmationAll: {
+    id: 'course.surveys.ResponseIndex.RemindButton.confirmationAll',
+    defaultMessage:
+      'Send emails to all students (incl. phantom) who have not completed the survey?',
```
Comment (zhuhanming) [inline client/app/bundles/course/survey/pages/ResponseIndex/RemindButton.jsx:28]:
```suggestion
      'Send emails to all students (including phantoms) who have not completed the survey?',
```

**Review by zhuhanming — APPROVED** [review]
Awesome, looks good to me! See if you're ok with my suggested changes, can just commit them if good.

**Pull request merged**